polyseam/cndi
# upgrade deno version to `1.38.0`

# upgrade to v1.38.0

CNDI is currently using deno v1.36.4. Breaking changes were introduced in v1.37.0 that must be fixed before we upgrade deno beyond 1.36.4.

# testing steps

1. checkout this branch
2. run `deno upgrade`
3. run `deno task test`
4. ensure tests pass

# Code of Conduct

By submitting this Pull Request, you agree to follow our
[Code of Conduct](https://github.com/polyseam/cndi/blob/main/CODE_OF_CONDUCT.md)


# Notes (Optional)
